commands/login.go

```diff
@@ -128,6 +147,10 @@ func promptAuthenticationMethod() {
 				"Authentication token (can be found under %s)",
 				colorizer.Bold(colorizer.Green("https://dashboard.tenderly.co/account/authorization")),
 			),
+			colorizer.Sprintf(
+				"Access key (can be generated on %s)",
```

Comment:
`Access key can be generated at https://dashboard.tenderly.co/account/authorization`